vendor/github.com/vmware/photon-controller-go-sdk/photon/client.go

```diff
@@ -142,8 +142,8 @@ func NewClient(endpoint string, options *ClientOptions, logger *log.Logger) (c *
 	}
 
 	restClient := &restClient{
-		httpClient:                &http.Client{Transport: tr},
-		logger:                    logger,
+		httpClient: &http.Client{Transport: tr},
+		logger:     logger,
```

Comment:
@stevekuznetsov @deads2k These go-fmt changes in github.com/vmware/photon were previously added when they were not supposed to.